frontendActions/src/main/resources/actions/ResourceBundle.properties

```diff
@@ -10,13 +10,17 @@ action.GitMachete.RebaseSelectedBranchOntoParentAction.GitMacheteContextMenu.tex
 action.GitMachete.RebaseSelectedBranchOntoParentAction.description=Rebase this branch onto parent
 
 action.GitMachete.PullSelectedBranchAction.text=Pull Selected Branch
-action.GitMachete.PullSelectedBranchAction.GitMacheteContextMenu.text=_Pull Branch
+action.GitMachete.PullSelectedBranchAction.GitMacheteContextMenu.text=P_ull Branch
 action.GitMachete.PullSelectedBranchAction.description=Pull this branch
 
 action.GitMachete.PushSelectedBranchAction.text=Push Selected Branch...
 action.GitMachete.PushSelectedBranchAction.GitMacheteContextMenu.text=_Push Branch...
 action.GitMachete.PushSelectedBranchAction.description=Push this branch using push dialog
 
+action.GitMachete.MergeSelectedBranchIntoParentAction.text=Merge Selected Branch Into Parent
```

Comment:
Hmm how about renaming that to `Fast forward parent to match this/current/selected/etc. branch`?

Reply:
Coz `merge X into Y`:
1. IMO requires more thinking about what branch is changed and what branch remains unchanged
1. `merge` for most people suggests a non-fast-forward merge, where an actual  merge commit is created

Reply:
If you agree, pls cascade this change into other affected strings/names in the codebase

Reply:
Very good point. It's a perfect idea to reflect that it is not some gitish merge. It's our, cool, nocheckout-requiring, fast forward merge.

We could do a similar thing to the pull action. WDYT?

Reply:
Hmmm what would you suggest for pull action then? :thinking:

Reply:
Fast forward current/selected branch to match remote?

Reply:
`Fast forward this/current/selected/etc. (local?) branch to match remote`?

Reply:
Hmm the problem I see with `Fast forward ... (local) branch to match remote`:  that's a bit ambiguous; it's not clear from this phrasing whether the fetch happens first or not, as opposed to e.g `Pull (fast-forward only) this branch`, where it's clear that the fetch does happen indeed

Reply:
Resolved to use `Pull (fast-forward only)`